polylogue/util.py

```
@@ -196,6 +199,18 @@ def update_conversation_state(
 
 
+def slugify_title(value: str) -> str:
+    if not value:
+        return ""
+    normalized = unicodedata.normalize("NFKD", value)
+    ascii_text = normalized.encode("ascii", "ignore").decode("ascii")
+    ascii_text = ascii_text.lower()
+    slug = re.sub(r"[^a-z0-9]+", "-", ascii_text)
+    slug = slug.strip("-")
+    slug = re.sub(r"-+", "-", slug)
+    return slug
```

Ruff (0.15.9)

[warning] Unnecessary assignment to `slug` before `return` statement

Remove unnecessary assignment

(RET504)